spec/initializers/track_setting_changes_spec.rb

```diff
@@ -43,4 +43,54 @@
       expect(Reviewable.min_score_for_priority(:low)).not_to eq(new_threshold)
     end
   end
+
+  describe "#title and site_description" do
+    before do
+      general_category = Fabricate(:category, name: "General")
+      SiteSetting.general_category_id = general_category.id
+      SeedData::Topics.with_default_locale.create(site_setting_names: ["welcome_topic_id"])
+    end
+
+    it "updates the welcome topic when title changes" do
+      SiteSetting.title = "foobar"
```

Comment:
Maybe this should be a random value (`SecureRandom.alphanumeric`) or a very unique string so that we're sure the tests we do later won't match something else?